src/FormCollection/Resources/views/form_theme_div.html.twig

```diff
@@ -0,0 +1,52 @@
+{%- block button_add -%}
+    {%- set attrDataAction = (attr['data-action']|default('') ~ ' ' ~ controllerName ~ '#add')|trim -%}
+    <button data-action="{{ attrDataAction }}"
```

Comment:
Would be better if we can use the exist `button_widget` block:

https://github.com/symfony/symfony/blob/41697e78a158156952f8c445512f11bd58f90958/src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig#L224

Maybe we can use in the Form Builder the `ButtonType` and then give it also a special `block_prefix` so we can go with `collection_button_add_widget` to overwrite it. 

So we could go with:

```php
'button_add_options => [
    'label' => 'app.button_add',
    'label_translation_parameters' => [],
    'attr' => ['class' => '...']
],
```

This make it possible to use any of the exist button options and would be easier for theming also for future themes.

Reply:
I made the changes based on your feedback.
Nexrt, the modifications on the buttons.

Reply:
Yea, this is interesting. To accomplish this, I think it would be very similar to how `prototype` is done, which is sort of a "fake field" that is added to the form... without being a real field.

You can see how the `prototype` is created in `buildForm()`... but not actually added to the field: https://github.com/symfony/symfony/blob/a05702965e9fc0de7ce77d1641e772e2fbfe9911/src/Symfony/Component/Form/Extension/Core/Type/CollectionType.php#L39-L40

Later, it's added as a view variable: https://github.com/symfony/symfony/blob/a05702965e9fc0de7ce77d1641e772e2fbfe9911/src/Symfony/Component/Form/Extension/Core/Type/CollectionType.php#L91-L101

It would be interesting to do the same thing with both buttons. We would pass `button_add_options` directly to the  `$builder->create()` call - e.g. how `prototype_options` are passed here: https://github.com/symfony/symfony/blob/a05702965e9fc0de7ce77d1641e772e2fbfe9911/src/Symfony/Component/Form/Extension/Core/Type/CollectionType.php#L33